pytorch_blade/benchmark/detectron2/test_d2_benchmark.sh

```diff
@@ -21,4 +22,4 @@ python3 run_blade.py --model testMaskRCNNFPN_pproc
 python3 run_blade.py --model testRetinaNet
 python3 run_blade.py --model testRetinaNet_scripted
 
-cd ..
+cd $oldpwd
```

Comment:
Nit: `cd -` will do the trick

Reply:
`cd -` only step back to the directory used before. But, here we would like to back a directory that executes this script.

Reply:
I see the entire script, if we only cd into a dir once, `cd -` is enough.